components/kyma-environment-broker/internal/provisioner/client.go

```diff
@@ -25,16 +30,34 @@ type Client interface {
 }
 
 type client struct {
-	graphQLClient *graphql.Client
+	graphQLClient *gcli.Client
 	queryProvider queryProvider
 	graphqlizer   Graphqlizer
+
+	queryDumping bool
 }
 
-func NewProvisionerClient(endpoint string, queryLogging bool) Client {
+func NewProvisionerClient(endpoint string, queryDumping bool) Client {
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
```

Comment:
we do not want to make that configurable?

Reply:
It seems we don't need this flag